### PR TITLE
Fix column sorting in the grade table

### PR DESCRIPTION
**Pull request by reillz10**

### CODE REVIEW

File: src/data/actions/utils.js [hunk 1]
```diff
@@ -25,39 +26,65 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
 
 const sortNumerically = (colKey, direction) => {
   function sortNumAsc(gradeRowA, gradeRowB) {
-    if (gradeRowA[colKey] < gradeRowB[colKey]) {
+    let a;
```
Comment (schenedx) [inline src/data/actions/utils.js]: Can you make sure you name the variables what they are? I don't like the meaningless variable names unless they are absolutely meaningless

Comment (reillz10) [reply src/data/actions/utils.js]: its a common pattern for sorting. We don't actually know what a or b is. It all depends on what is passed in

Comment (reillz10) [reply src/data/actions/utils.js]: I can call it `valueAtGradeRowA` but I feel like that is just as arbitrary as `a`

File: src/data/actions/utils.js [hunk 2]
```diff
@@ -25,39 +26,65 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
...
-    if (gradeRowA[colKey] > gradeRowB[colKey]) {
+    if (a > b) {
       return 1;
     }
     return 0;
   }
 
   function sortNumDesc(gradeRowA, gradeRowB) {
-    if (gradeRowA[colKey] < gradeRowB[colKey]) {
+    let a;
```
Comment (iloveagent57) [inline src/data/actions/utils.js]: Everything from here to the `if (a < b)` statement is exactly the same as the lines above.  It's probably worth extracting this to a shared function.

File: src/data/actions/utils.js [hunk 3]
```diff
@@ -25,39 +26,65 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
...
+    }
+
+    if (a < b) {
       return -1;
     }
-    if (gradeRowA[colKey] > gradeRowB[colKey]) {
+    if (a > b) {
       return 1;
     }
     return 0;
```
Comment (iloveagent57) [inline src/data/actions/utils.js]: Can't you just return `a - b` without checkout for equality?

File: src/data/actions/utils.js [hunk 4]
```diff
@@ -25,39 +26,65 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
...
+    }
+
+    if (a < b) {
       return 1;
     }
-    if (gradeRowA[colKey] > gradeRowB[colKey]) {
+    if (a > b) {
       return -1;
     }
     return 0;
```
Comment (iloveagent57) [inline src/data/actions/utils.js]: ...and return `b - a` here?

File: src/data/actions/utils.js (hunk 3 above)
Comment (reillz10) [reply src/data/actions/utils.js]: Yes I can for the number based sorts.

**Pull request merged**